### PR TITLE
Add backport 6.0 label to automated PRs.

### PR DESCRIPTION
**Pull request by advait-zx**
### Summary of Changes
This pull request updates the `.github/workflows/sub_package_update.yml` file to automatically add the "backport 6.0" label to pull requests created by the workflow. This label helps to ensure that any necessary updates are backported to the relevant branches.

### Issue Addressed
No specific issue was raised for this change, but it addresses the need to streamline the backporting process for template updates.

### Motivation and Context
Automating the addition of the "backport 6.0" label will ensure that template changes are consistently backported, reducing the risk of missing important updates and improving overall workflow efficiency.

### Additional Notes
- The workflow now includes the `labels` input in the `peter-evans/create-pull-request` action.
- Please review the changes and let me know if any adjustments are needed.

### CODE REVIEW

File: .github/workflows/sub_package_update.yml [hunk 1]
```diff
@@ -75,5 +75,10 @@ jobs:
           branch-suffix: timestamp
           title: ${{ matrix.title }}
           body: |
+            This is an autogenerated PR, which will ${{ matrix.body }}.
+            [Cruft](https://cruft.github.io/cruft/) has detected updates from the Package Template.
```
Comment (Cadair) [inline .github/workflows/sub_package_update.yml:79]: you don't need to duplicate these lines. you should add the labels line at the same level of indentation as `body` at the end of the file.

**Review by Cadair — COMMENTED** [review]
Thanks for the PR @advait-zx

File: .github/workflows/sub_package_update.yml (hunk 1 above)
Comment (advait-zx) [reply .github/workflows/sub_package_update.yml]: I have removed duplicated lines.

**Review by Cadair — APPROVED** [review]

**Comment by Cadair** [comment]
Thanks @advait-zx

**Pull request merged**